tests/cmd/start-in-future.stderr

```diff
@@ -1 +1 @@
-ERROR: start date should be on or before current date, got start date request: 9999-01-01 and current date is [..]
+ERROR: start date should be on or before current date, got start date request: 9999-01-01 and current date is 2024-04-15
```

Comment:
Looks like this snuck in. Can you remove this change? Otherwise, tests will fail.